File: VBDotNet/src/Bear.vb
Namespace ClassZoo
    Public Class Bear 
        Inherits Animal
        Sub New(InputName As String, InputID As Integer, InputWeight As Single, InputHeight As Single)
            MyBase.New(InputName, InputID, InputWeight, InputHeight)
        End Sub

        Public Overrides Sub PrintSpecies()
            Console.WriteLine("Animal Species: Bear")
        End Sub 
        Public Overrides Sub AnimalSound()
            Console.WriteLine(Me.GetName() + " growls")
        End Sub
    End Class
End Namespace

File: C_Sharp/src/Bear.cs
namespace ClassZoo;

public class Bear : Animal 
{
    public Bear(string inputName, int inputID, float inputWeight, float inputHeight) 
    : base(inputName, inputID, inputWeight, inputHeight)
    {
    }
    public override void PrintSpecies()
    {
        Console.WriteLine("Animal Species: Bear");
    }
    public override void AnimalSound()
    {
        Console.WriteLine($"{this.GetName() + " growls"}");
    }
}